sound/soc/sof/sof-priv.h

```diff
@@ -415,6 +415,8 @@ struct snd_sof_dev {
 	u32 host_offset;
 	u32 dtrace_is_enabled;
 	u32 dtrace_error;
+	u32 dtrace_eof;		/* trace EOF pending */
```

Comment:
dtrace_eof is set at the init (twice), so I am confused if we really have a 'trace EOF pending' during regular execution. Something's not quite self-explanatory here.